odysseus/halow_setup/sources/newracom-blacklist.conf

```diff
@@ -0,0 +1,2 @@
+blacklist brcmfmac
```

Comment:
What is the purpose of this blacklist? Is it for specific functionalities of the driver we don't care about?

Reply:
Ill go step by step and describe all source files in the README.  That simply blacklists braodcom wifi and utility functions as they conflict with the driver we load.  There is a way around this teledatics describes but it is not well documented.